docs/source/workflows/mcp/mcp-service-account-auth.md

```
@@ -0,0 +1,275 @@
+    _type: mcp_service_account
+    client_id: ${CLIENT_ID}
+    client_secret: ${CLIENT_SECRET}
+    token_url: ${TOKEN_URL}
+    scopes: "service.scope"
+    token_prefix: service_account_ssa
+    service_token: ${SERVICE_TOKEN}
+    service_token_header: X-Service-Account-Token
+```
+
+This produces the following headers:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)